controllers/postComments.controllers.js

```diff
@@ -0,0 +1,23 @@
+const { fetchPostComments } = require('../Models/postComments.models')
+
+
+exports.postComments = (req, res) =>
+{
+    const { username, body } = req.body
+    const { article_id} = req.params
+
+    fetchPostComments(username, body, article_id).then((newComment) =>
+    {
+        res.status(201).send(newComment)
```

Comment:
The new comment is supposed to be sent back in an object, on a key of newComment. Currently you are sending back an object in an array.